Cargo.toml

```diff
@@ -32,10 +31,12 @@ lru = "0.7.5"
 zeroize = { version = "1.5.5", features = [ "zeroize_derive" ] }
 
 aws-sdk-kms = { version = "0.10.1", optional = true }
+aws-config = { version = "0.10.1", optional = true }
 
 [features]
-# Enable the KMS Key Provider
-aws-kms = ["dep:aws-sdk-kms"]
+default = ["aws-kms", "cache"]
```

Comment:
Making `aws-kms` and `cache` default features so they get tested, plus they're only don't work in WASM.